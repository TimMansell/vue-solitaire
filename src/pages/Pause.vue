<template>
  <GameOverlay center-content show-logo data-test="pause">
    <template #title> Game Paused</template>
    <template #msg>
      <GameSummary />
    </template>
    <template #buttons>
      <ResumeGameButton />
    </template>
  </GameOverlay>
</template>

<script>
import GameOverlay from '@/components/GameOverlay.vue';
import GameSummary from '@/components/GameSummary.vue';
import ResumeGameButton from '@/components/ResumeGameButton.vue';

export default {
  name: 'PausePage',
  components: {
    GameOverlay,
    ResumeGameButton,
    GameSummary,
  },
};
</script>
