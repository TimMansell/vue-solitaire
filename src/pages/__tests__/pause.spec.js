import { shallowMount } from '@vue/test-utils';
import Pause from '@/pages/Pause.vue';

describe('Pause.vue', () => {
  it('renders the component without crashing', () => {
    const wrapper = shallowMount(Pause);

    expect(wrapper.isVisible()).toBe(true);
  });
});
