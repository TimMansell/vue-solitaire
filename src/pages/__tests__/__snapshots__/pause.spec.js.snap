// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pause.vue matches snapshot 1`] = `
<gameoverlay-stub centercontent="true" showlogo="true" data-test="pause"> Game Paused<gamesummary-stub></gamesummary-stub>
  <resumegamebutton-stub></resumegamebutton-stub>
</gameoverlay-stub>
`;
